.github/workflows/aks.yml

```diff
@@ -71,6 +68,10 @@ jobs:
       - name: Lint Epinio
         run: make lint
 
+      - name: Clean all
+        if: always()
```

Comment:
I see [in examples](https://github.com/colpal/actions-clean#usage) curlies being used `${{}}`, are they needed here?

Reply:
We have without them in the UI tests and according to the [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif) in that case (no variable) they can be omitted. But I can add them if your prefer, for consistency with the other steps.